kalite/settings/base.py

```diff
@@ -314,8 +314,22 @@
 USE_L10N = getattr(local_settings, "USE_L10N", False)
 
 # Make this unique, and don't share it with anybody.
-SECRET_KEY = getattr(local_settings, "SECRET_KEY",
-                     "8qq-!fa$92i=s1gjjitd&%s@4%ka9lj+=@n7a&fzjpwu%3kd#u")
+SECRET_KEY_FILE = os.path.join(USER_DATA_ROOT, "secretkey.txt")
```

Comment:
re: file might not exist -- that's why we create the file if it doesn't exist. It's also done in `USER_DATA_ROOT` which we assume is always writeable by the server.